### PR TITLE
zfs: update to 0.8.4.

### PR DESCRIPTION
**Pull request by ahesford**

### CODE REVIEW

File: srcpkgs/zfs/patches/dkms.patch [hunk 1]
```diff
@@ -1,44 +0,0 @@
-# Patch from https://aur.archlinux.org/packages/zfs-dkms/
-# Avoids recompiling ZFS userland utilities with DKMS rebuilds
-# Also short-circuits a useless test for Debian in the DKMS source
```
Comment (Vaelatern) [inline srcpkgs/zfs/patches/dkms.patch:3]: Why did you remove this patch?

Comment (ahesford) [reply srcpkgs/zfs/patches/dkms.patch]: By mistake; I meant to remove only the patch for linux5.6 that was made obsolete by upstream changes. The latest push restores this patch.

**Comment by zdykstra** [comment]
I've tested this locally - works as a root filesystem without any issues.
